src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -2028,6 +2046,25 @@ TEST_F(ApplicationManagerImplTest, AddAndRemoveQueryAppDevice_SUCCESS) {
   EXPECT_FALSE(app_manager_impl_->IsAppsQueriedFrom(device_handle));
 }
 
+TEST_F(
+    ApplicationManagerImplTest,
+    RequestForInterfacesAvailability_AllRequestsWillBeSuccessfullyRequested) {
+  std::vector<hmi_apis::FunctionID::eType> expected_requests{
+      hmi_apis::FunctionID::VehicleInfo_IsReady,
+      hmi_apis::FunctionID::VR_IsReady,
+      hmi_apis::FunctionID::TTS_IsReady,
+      hmi_apis::FunctionID::UI_IsReady,
+      hmi_apis::FunctionID::RC_IsReady};
+
+  for (auto request : expected_requests) {
+    SetUpCreateModuleInfoSOExpectation(request);
+    EXPECT_CALL(*mock_rpc_service_,
+                ManageHMICommand(HMIResultCodeIs(request), _));
```

Comment:
@ydementieiev Add secon parameter please for clarity.

Reply:
@AByzhynar [added](https://github.com/LuxoftSDL/sdl_core/pull/38/commits/0a5924e2dc93b929033b4e10a282660445f7580e)